### ISSUE TITLE
installation problem

### ISSUE THREAD

**Issue by jscottlex** [5ha0/fortools#1]
Python 3.8.10 (default, Nov 14 2022, 12:59:47) 
[GCC 9.4.0] on linux

sudo pip install fortools
[sudo] password for awow:        
Collecting fortools
  Downloading fortools-0.0.10-py3-none-any.whl (65 kB)
     |████████████████████████████████| 65 kB 276 kB/s 
Requirement already satisfied: numpy in /usr/lib/python3/dist-packages (from fortools) (1.17.4)
Collecting olefile
  Downloading olefile-0.46.zip (112 kB)
     |████████████████████████████████| 112 kB 988 kB/s 
Collecting xmltodict
  Downloading xmltodict-0.13.0-py2.py3-none-any.whl (10.0 kB)
Collecting libevtx-python
  Downloading libevtx-python-20221101.tar.gz (2.1 MB)
     |████████████████████████████████| 2.1 MB 3.9 MB/s 
Requirement already satisfied: matplotlib in /usr/lib/python3/dist-packages (from fortools) (3.1.2)
Collecting PyPDF2
  Downloading pypdf2-2.12.1-py3-none-any.whl (222 kB)
     |████████████████████████████████| 222 kB 1.8 MB/s 
Collecting libesedb-python
  Downloading libesedb-python-20220806.tar.gz (2.0 MB)
     |████████████████████████████████| 2.0 MB 6.8 MB/s 
Collecting python-registry
  Downloading python_registry-1.3.1-py3-none-any.whl (23 kB)
Collecting image
  Downloading image-1.5.33.tar.gz (15 kB)
Requirement already satisfied: chardet in /usr/lib/python3/dist-packages (from fortools) (3.0.4)
Requirement already satisfied: python-magic in /usr/lib/python3/dist-packages (from fortools) (0.4.16)
Collecting bitstring
  Downloading bitstring-4.0.1-py3-none-any.whl (45 kB)
     |████████████████████████████████| 45 kB 443 kB/s 
Collecting pytsk3
  Downloading pytsk3-20211111.tar.gz (3.2 MB)
     |████████████████████████████████| 3.2 MB 7.3 MB/s 
ERROR: Could not find a version that satisfies the requirement python-magic-bin (from fortools) (from versions: none)
ERROR: No matching distribution found for python-magic-bin (from fortools)

**Comment by jscottlex** [comment]
i tried 
sudo pip install python-magic
Requirement already satisfied: python-magic in /usr/lib/python3/dist-packages (0.4.16)

**Comment by green339** [comment]
First of all, I apologize for the delay in answering.
The fortools is a project that has been temporarily suspended due to personal reasons of the members.

Looking at the contents of the issue, it seems that the problem occurred in the "python-magic-bin" part, but did you install it in an environment other than Windows? There is a possibility that this part will not work in other environments. Currently, fortsools have been implemented for Windows and further implementation has stopped.
We have plans to add the development environment in the future, but there is no exact schedule.

Thank you for watching Fortools with interest.
We'll let you know again if it's updated. :)

**Comment by jscottlex** [comment]
Ok, thank you. I attempted to install on Linux Mint.Rev J. Scott Martin, P.E.On Jan 6, 2023, at 8:55 AM, maplejh ***@***.***> wrote:﻿
First of all, I apologize for the delay in answering.
The fortools is a project that has been temporarily suspended due to personal reasons of the members.
Looking at the contents of the issue, it seems that the problem occurred in the "python-magic-bin" part, but did you install it in an environment other than Windows? There is a possibility that this part will not work in other environments. Currently, fortsools have been implemented for Windows and further implementation has stopped.
We have plans to add the development environment in the future, but there is no exact schedule.
Thank you for watching Fortools with interest.
We'll let you know again if it's updated. :)
